### PR TITLE
✨ focus scanning on customer workloads

### PR DESCRIPTION
**Pull request by slntopp**

### CODE REVIEW

File: controllers/resource_monitor/resource_monitor_controller.go [hunk 1]
```diff
@@ -82,8 +84,23 @@ func (r *ResourceMonitorController) Start(ctx context.Context) error {
 }
 
 func (r *ResourceMonitorController) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	obj := r.createRes()
```
Comment (imilchev) [inline controllers/resource_monitor/resource_monitor_controller.go:87]: this code looks correct to me. Please add some test cases in `controllers/resource_monitor/resource_monitor_controller_test.go`. Everything there should be already mocked, so it should be fairly simple to verify that we aren't add resources to the debouncer that have a parent we don't care about

Comment (slntopp) [reply controllers/resource_monitor/resource_monitor_controller.go:87]: Updated the tests ✨

**Comment by imilchev** [comment]
You will need to update the integration tests since they verify the workloads that end up in the mondoo platform. Since this now changed, the tests need to be updated to expect the correct workloads

**Comment by slntopp** [comment]
@imilchev yep, working on it rn

**Review by imilchev — APPROVED** [review]
LGTM

**Pull request merged**